models/meta/re_data_monitored.sql

```diff
@@ -1,15 +1,10 @@
 {{
     config(
         materialized='table',
-        unique_key = 'table_name'
+        unique_key = 'table_name',
+        post_hook="{% if execute %}{{ pub_insert_into_re_data_monitored() }}{% endif %}"
```

Comment:
I see `post_hook` is run after a model, seed or snapshot is built. Would this handle duplicate entries on multiple runs? Since the macro does an insert into the table.

Reply:
This model is regenerated every time re_data runs, so double inserts shouldn't be an issue..